system/main/cron.php

```diff
@@ -4,17 +4,27 @@
  * execute cron job for all modules
  */
 function do_cron() {
-  global $config;
+  global $files_dir;
   ct_log("Cron-Job started.", 2, -1, 'cron');
-  
-  $btns = churchcore_getModulesSorted(false, false);
-  foreach ($btns as $key) {
+
+  //delete temporary files (and sessions) older than one day
+  $tempDir = $files_dir . '/tmp/';
+  foreach (array_slice(scandir($tempDir), 2) as $file) {
+    $path = $tempDir . $file;
+    if (is_file($path) && filemtime($path) < time() - (3600 * 24)) {
```

Comment:
I hate constants, seriously. They tend to screw up everything as the project grows and imho hardly help anything in such a case. I use only class-bound constants in autoloaded classes and else avoid them wherever possible.
Apart from my opinion: 24 hours is just some figure which seems suitable to me. I don't see any technical reason for this value to be changed in the near future. Do you see one?